tests/integration/tests/test_util/harness/juju.py

```diff
@@ -53,7 +53,10 @@ def __init__(self):
                 self.constraints,
             )
 
-    def new_instance(self) -> Instance:
+    def new_instance(self, dualstack: bool = False) -> Instance:
+        if dualstack:
+            raise HarnessError("Dualstack is currently not supported by Juju harness")
```

Comment:
I may have missed the discussion, why does dualstack testing only work on lxd?

Reply:
Basically, we just don't support that for mulitpass etc because we don't need it right now and it is not straight-forward to implement.